Gruntfile.js

```diff
@@ -29,15 +37,62 @@
         }]
       }
     },
+    aws_s3: {
+      options: {
+        accessKeyId: aws.AWSAccessKeyID,
+        secretAccessKey: aws.AWSSecretKey,
+        region: 'us-east-1'
```

Comment:
`eu-west-1` might be better for us, it's the region where most of our content lives